components/profile-controller/controllers/profile_controller.go

```diff
@@ -55,6 +58,9 @@ const USER = "user"
 const ROLE = "role"
 const ADMIN = "admin"
 
+// External input for labels to be added to namespace.
+const NAMESPACE_LABELS_PROPERTIES = "/etc/config/labels/namespace-labels.properties"
```

Comment:
it's not a good practice to hard-code config file location, it's reasonable to specify a default location, but the default location should typically be overridable from a command line argument.

Reply:
Make sense, I changed it to a CLI argument with default value. Thank you!